tests/testthat/_snaps/cast.md

```diff
@@ -12,15 +12,15 @@
       vec_cast(1, "", x_arg = "foo")
     Condition
       Error:
-      ! Can't convert `foo` <double> to <character>.
+      ! Can't convert `foo` <double> to match type of `""` <character>.
 
 # cast errors create helpful messages (#57, #225)
 
     Code
       vec_cast(1.5, 10L)
     Condition
       Error:
-      ! Can't convert from <double> to <integer> due to loss of precision.
+      ! Can't convert from `1.5` <double> to `10L` <integer> due to loss of precision.
```

Comment:
I think what I don't love about this is that the message makes me think I am trying to cast to the _value_ of `10L`, when really I am trying to cast to the _type_ of `10L` which is <integer>

Reply:
Oh it looks like "to match type of `10`" is used below, so may it is just the lossy cast message that needs an update?

Reply:
Maybe:

```
Can't convert `1.5` <double> to match type of `10L` <integer> due to loss of precision.
```

Which would be more consistent with the general cast error anyways